### DISCUSSION TITLE
Show player range and scanning outlines for neutral stars

### DISCUSSION THREAD

**Discussion by Skilly146** [Tsangares/stoned_ape_tools#64] [Feature Requests]
## What

Show the scanning range and hyperspace range visuals when the player selects a neutral star, just like when they click their own star.

## Why

Most of the time that someone is looking at a neutral star it is when they are imagining it as part of their empire, so why not give it the same tools? Instead of having to use the ruler the player can just automatically see what stars they can reach from the selected star and what routes they can take.

## How

Simply add a bit of code to detect when the selected star is unowned, and if so show the correct size pf scanning and hyperspace visual using the built in method I assume exists.
```
if selectedStar.isUnowned:
        showRangeVisual(selectedStar)
        showScanningVisual(selectedStar
```

## Problems

We just need to find the method the game uses for showing scanning and range visuals then we can easily just activate it for neutral stars as if they were the players

**Comment by Tsangares** [comment]
I have a question if I understand the ultimate goal of this. If there exists an unowned star, when I select this star, the plugin conveys to me the user in some way the distances to the other players in the game. So it kinda can convey who **should** own which stars. Is that correct? 

Like stars within a level 1 jump from a star I own will obviously be mine but distant stars may be **contested** and that's what this feature would convey?

**Reply by Skilly146** [reply]
While that in itself is a very promising feature, mine is much simpler. All it does is show the hyperspace and scanning range visuals (the dotted and ridged circles around selected stars) for neutral star the same way they would look if you selected your own star

**Reply by Tsangares** [reply]
oh I see, that can be done.
